tests/schemas/api_v3/networkStatus.schema.js

```diff
@@ -55,7 +54,7 @@ const networkStatusSchema = {
 	unconfirmedTransactions: Joi.number().integer().min(0).required(),
 	genesis: Joi.object(genesisSchema).required(),
 	registeredModules: Joi.array().items(Joi.string()).required(),
-	moduleCommands: Joi.array().items(moduleCommandSchema).required(),
+	moduleCommands: Joi.array().items(Joi.string()).required(),
```

Comment:
Also, kindly remove `MODULE_COMMAND_ID` and `MODULE_COMMAND_NAME`. Ideally, `MODULE_COMMAND_ID` shouldn't be getting used elsewhere. `MODULE_COMMAND_NAME` can be replaced with `MODULE_COMMAND`.
```suggestion
	moduleCommands: Joi.array().items(Joi.string().pattern(regex.MODULE_COMMAND)).required(),
```

Reply:
Applied.

Reply:
The pattern check is still missing. Please update.
```suggestion
	moduleCommands: Joi.array().items(Joi.string().pattern(regex.MODULE_COMMAND)).required(),
```

Reply:
Applied.